receiver/opencensusreceiver/octrace/opencensus.go

```diff
@@ -160,7 +165,7 @@ func (ocr *Receiver) sendToNextConsumer(longLivedRPCCtx context.Context, traceda
 		err = ocr.nextConsumer.ConsumeTraceData(ctx, *tracedata)
```

Comment:
Bug: We did not pass the tags from the longLivedRPCCtx to the consumer so metrics recorded in the producer/exporter will not have them.